### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by li-s**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -129,13 +129,28 @@ This section describes some noteworthy details on how certain features are imple
...
+It then gets the filepath of the `Tag` before safely deleting it. Then, a new `Tag` is created with the filepath, and the new tag name.
+
 ### Adding of Tags: TagCommand
 
 [TagCommand](https://github.com/AY2021S1-CS2103T-F12-1/tp/blob/master/src/main/java/seedu/address/logic/commands/TagCommand.java) 
 adds a new `Tag` to `AddressBook` if the tag's `TagName` is not a duplicate and the tag's `FileAddress`
 is pointing to a valid file.
 
-TagCommand checks if the file is present using `java.io.File.exists()`.
+TagCommand checks the presence of the `Tag` using `java.io.File.exists()`.
```
Comment (luo-git) [inline docs/DeveloperGuide.md]: This edit is not correct? TagCommand do check if the file is present using `java.io.File.exists()`.

**Review by luo-git — CHANGES_REQUESTED** [review]
Some minor changes requried.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (li-s) [reply docs/DeveloperGuide.md]: dafaq i didnt touch that line

Comment (luo-git) [reply docs/DeveloperGuide.md]: What?

**Pull request merged**